service/src/main/java/bio/terra/landingzone/library/landingzones/management/ResourcesDeleteManager.java

```diff
@@ -161,11 +162,15 @@ private GenericResource deleteResource(ResourceToDelete resourceToDelete) {
               });
     }
 
-    armManagers
-        .azureResourceManager()
-        .genericResources()
-        .deleteById(resourceToDelete.resource().id());
-
+    if (resourceToDelete.resource().id().contains("virtualNetworkLinks")) {
+      // this is temporary solution since generic deletion is failing for this type of resource
```

Comment:
Please drop a tech debt remediation ticket for this if one is not already in place.